build.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -e
+
+START_DIR=$PWD
```

Comment:
question: do we need a cd $START_DIR here?

Reply:
I think I can remove that line to just cd into the pack dir